### PR TITLE
[TIMOB-20431]: Remove NonValidatingTrustManager.

### PR DESCRIPTION
**Pull request by hieupham007**
Testing steps in JIRA. PLEASE DO NOT MERGE YET.

### CODE REVIEW

**Comment by ashcoding** [comment]
Code reviewed. Functionally tested. **Passed**. Would require documentation to be added.

File: android/modules/network/src/java/ti/modules/titanium/network/HTTPClientProxy.java [hunk 1]
```diff
@@ -217,7 +217,7 @@ public void setAutoRedirect(boolean value)
 	@Kroll.getProperty @Kroll.method
 	public boolean getValidatesSecureCertificate()
```
Comment (FokkeZB) [inline android/modules/network/src/java/ti/modules/titanium/network/HTTPClientProxy.java]: Why don't we just remove this method?

**Comment by adoblas** [comment]
+1

**Comment by hieupham007** [comment]
Closing since this issue doesn't affect 5.2.0

**Pull request closed without merging**